api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -29,6 +31,10 @@
 @Handler(supports = { Visit.class }, order = 50)
 public class VisitValidator extends BaseCustomizableValidator implements Validator {
 	
+	private static final double ESTIMATED_BIRTHDATE_ERROR_MARGIN = -0.5;
+	
+	private static final int ESTIMATED_BIRTHDATE_ERROR_MARGIN_MINIMUM = -1;
```

Comment:
rename to ESTIMATED_BIRTHDATE_ERROR_MARGIN_MINIMUM_YEARS

so its clear its unit is years